lua/SS13_base.lua

```diff
@@ -19,12 +19,46 @@ function SS13.get_runner_client()
 	return dm.global_vars.GLOB.directory[SS13.get_runner_ckey()]
 end
 
+SS13.ispath = dm.global_procs._ispath
 SS13.type = dm.global_procs._text2path
 
 function SS13.istype(thing, type)
 	return dm.global_procs._istype(thing, SS13.type(type)) == 1
 end
 
+function SS13.typecacheof(string_types)
+	local types = {}
+	for _, path in pairs(string_types) do
```

Comment:
```suggestion
	for _, path in string_types do
```

Don't need this in Luau